js/tests/README.md

```diff
@@ -0,0 +1,44 @@
+## How does Bootstrap's test suite work?
+
+Bootstrap uses [QUnit](http://api.qunitjs.com/), a powerful, easy-to-use JavaScript unit test framework. Each plugin has a file dedicated to its tests in `unit/[plugin-name].js`.
+
+* `unit/` contains the unit test files for each Bootstrap plugin.
+* `vendor/` contains third-party testing-related code for the browser environment (jQuery and QUnit)
+* `visual/` contains visual test code which is manually evaluated.
+
+To run the unit test suite via [PhantomJS](http://phantomjs.org/), run `grunt test-js`.
+To run the unit test suite via a real web browser, open [`index.html`](https://github.com/twbs/bootstrap/blob/master/js/tests/index.html) in the browser.
```

Comment:
I think it might be confusing to hyperlink this. Don't want folks to confuse the GitHub file viewer page with the actual local HTML page.